### PR TITLE
FOUR-8242

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
[FOUR-8242](https://processmaker.atlassian.net/browse/FOUR-8242)

## Solution
N/A

## How to Test
Describe how to test that this solution works.

## Related Tickets & Packages
[- Link to any related FOUR tickets, PRDs, or packages](https://processmaker.atlassian.net/browse/FOUR-8242)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy

[FOUR-8242]: https://processmaker.atlassian.net/browse/FOUR-8242?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

.

### CODE REVIEW

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-e050779d0e.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-e050779d0e.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-e050779d0e.eng.processmaker.net

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-e050779d0e.eng.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-e050779d0e.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-e050779d0e.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-e050779d0e.eng.processmaker.net

File: ProcessMaker/Events/CustomizeUiUpdated.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    private array $data;
+    private array $changes;
+    private array $original;
+
+    /**
+     * Create a new event instance.
+     *
+     * @return void
+     */
+    public function __construct(array $original, array $changes, string $update_at)
```
Comment (pmPaulis) [inline ProcessMaker/Events/CustomizeUiUpdated.php:22]: $update_at please use a lowerCamelCase $updatedAt

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-e050779d0e.eng.processmaker.net

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Review by pmPaulis — APPROVED** [review]

**Pull request merged**